libeos-updater-util/flatpak.c

```diff
@@ -289,14 +295,7 @@ flatpak_remote_ref_action_from_json_node (JsonNode *node,
   if (serial_node == NULL ||
       !JSON_NODE_HOLDS_VALUE (serial_node) ||
       json_node_get_value_type (serial_node) != G_TYPE_INT64)
-    {
-      g_autofree gchar *node_str = json_node_to_string (node);
-      g_set_error (error,
-                   EOS_UPDATER_ERROR,
-                   EOS_UPDATER_ERROR_MALFORMED_AUTOINSTALL_SPEC,
-                   "Expected a 'serial' member in the autoinstall spec (at %s)", node_str);
-      return NULL;
-    }
+    return NULL;
```

Comment:
This will result in returning `NULL` but with no error set, I think. Is that OK?

Reply:
Ah yes, we'll need to set the error - must have overlooked it.